guides/quick-start.md

```diff
@@ -0,0 +1,61 @@
+## At a glance
+
+There are two main routes to getting started with Mailchimp Open Commerce: installing the platform on your local computer or on a server. This guide will focus on the former, which will allow you to explore the main features of Open Commerce.
+
+In this guide, we’ll set up a full local instance of the Open Commerce platform, including core plugins provided by Mailchimp. We’ll walk through cloning and starting the Open Commerce development platform, registering an account, and creating your first shop. 
+
+## What you’ll need
+
+  * [Docker](https://www.docker.com/)
+
+  * [Docker Compose](https://docs.docker.com/compose/)
+
+  * Windows users: [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Docker for WSL](https://docs.docker.com/docker-for-windows/wsl/)
+
+## Clone and start the platform
+
+The simplest way to get an instance up and running on your local machine is by using the [development platform](https://github.com/reactioncommerce/reaction-development-platform), a set of `make` scripts that coordinates all of the different pieces of the Open Commerce software system. 
+
+To set up the Open Commerce development platform, navigate to a Unix or WSL directory and run:
+
+```bash
+git clone https://github.com/reactioncommerce/reaction-development-platform.git
+cd reaction-development-platform
+make
```

Comment:
@Akarshit is this correct? can you run `make` or it has to be `make init`?
CC @alexmarple

Reply:
It can be `make`. `make` does the same thing as `make init`